vimeo-deeplink/src/main/java/com/vimeo/android/deeplink/VimeoDeeplink.java

```diff
@@ -59,13 +63,17 @@
     private static final String EXPLORE = "/explore";
     private static final String FEED = "/feed";
     private static final String ME = "/me";
+    private static final String OFFLINE = "/offline";
     private static final String PLAYLISTS = "/playlists";
+    private static final String PURCHASES = "/purchases";
     private static final String UPLOAD = "/upload";
+    private static final String WATCH_LATER = "/watchlater";
 
     public static final String VIMEO_VIDEO_URI_PREFIX = "/videos/";
     public static final String VIMEO_USER_URI_PREFIX = "/users/";
     public static final String VIMEO_CATEGORY_URI_PREFIX = "/categories/";
     public static final String VIMEO_CHANNEL_URI_PREFIX = "/channels/";
+    public static final String VIMEO_ONDEMAND_URI_PREFIX = "/ondemand/";
```

Comment:
Ticket is wrong, and was created before enough investigation was done into on demand. Series and films have the same uri structure. For example, https://vimeo.com/ondemand/lonelyandhorny is a series and https://vimeo.com/ondemand/cartelland is a film. I will update the ticket to reflect.